service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -70,22 +79,105 @@ private void run3037(CaseDetails caseDetails) {
         caseDetails.setData(data);
     }
 
-    private void run3093(CaseDetails caseDetails) {
+    private void run3080(CaseDetails caseDetails) {
         CaseData caseData = getCaseData(caseDetails);
 
         List<Element<AdditionalApplicationsBundle>> additionalApplicationsBundle = caseData
             .getAdditionalApplicationsBundle();
 
-        if (caseData.getAdditionalApplicationsBundle().size() < 1) {
+        Element<AdditionalApplicationsBundle> firstBundle = additionalApplicationsBundle.get(0);
```

Comment:
this one still may fail if new application uploaded meanwhile, but thats ok, migration will fail